src/components/Login.js

```diff
@@ -1,19 +1,27 @@
 import React, { Component } from 'react'
+import { connect } from 'react-redux'
 import styled from 'styled-components'
-import LoginWithSlackButton from "./LoginWithSlackButton";
+import LoginWithSlackButton from './LoginWithSlackButton'
+import { login } from '../store/actions/auth.actions'
+import Button from './Button'
 
 class Login extends Component {
     render () {
         return (
             <Container>
                 <h4>Kto tam?</h4>
-                <LoginWithSlackButton/>
+                {/*<LoginWithSlackButton/>*/}
+                <Button onClick={this.props.login} name={'hipopotam'}>Hipopotam</Button>
```

Comment:
A Hipo button? 😆

Reply:
Yes, something wrong here? 😄 IMO this button is perfect 👌